cmd/query.go

```diff
@@ -104,13 +106,29 @@ func runQueryCmd(cmd *cobra.Command, args []string) {
 		// ensure client is closed
 		defer client.Close()
 
+		ctx := setupQueryCancel()
```

Comment:
can we create context here and just pass the cancel function into `setupQueryCancel`